rotkehlchen/chain/evm/proxies_inquirer.py

```diff
@@ -110,8 +111,7 @@ def get_accounts_having_proxy(self) -> dict[ChecksumEvmAddress, ChecksumEvmAddre
 
         with self.node_inquirer.database.conn.read_ctx() as cursor:
             accounts = self.node_inquirer.database.get_blockchain_accounts(cursor)
-        eth_accounts = accounts.eth
-        mapping = self.get_accounts_proxy(eth_accounts)
+        mapping = self.get_accounts_proxy(accounts.get(self.node_inquirer.blockchain))
```

Comment:
This was another bug I found. We always were taking eth accounts here irrespective of the evm chain type